### PR TITLE
Update bundler dependency

### PR DESCRIPTION
**Pull request by anero**

### CODE REVIEW

**Comment by anero** [comment]
This should resolve #289

**Review by zalt50 — CHANGES_REQUESTED** [review]
Juwelier itself should be removed from the dependence to prevent "infinite loop"

**Comment by anero** [comment]
@zalt50 what do you think of 96fecac489514d00fffba5526c8d2b1bb23d6ca1? Was the self dependency only included to allow using juwelier's rake tasks on the gem itself?

**Comment by zalt50** [comment]
@anero it works well!

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ gem "git", ">= 1.2.5"
 gem "nokogiri", ">= 1.5.10"
 gem "github_api"
 gem "highline", ">= 1.6.15"
-gem "bundler", ">= 1.0"
+gem "bundler", ">= 1.13"
```
Comment (flajann2) [inline Gemfile:8]: I think it should be a bit more general and made 
gem "bundler", ">= 1"
What are your thoughts on this?

**Review by flajann2 — APPROVED** [review]

**Pull request merged**